state/metricsmanager.go

```diff
@@ -89,22 +110,44 @@ func (st *State) getMetricsManager() (*MetricsManager, error) {
 	coll, closer := st.db().GetCollection(metricsManagerC)
 	defer closer()
 	var doc metricsManagerDoc
-	err := coll.FindId(metricsManagerKey).One(&doc)
+	err := coll.FindId(metricsManagerKey(st)).One(&doc)
 	if err == mgo.ErrNotFound {
 		return nil, errors.NotFoundf("metrics manager")
 	} else if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return &MetricsManager{st: st, doc: doc}, nil
+	collS, closerS := st.db().GetCollection(meterStatusC)
+	defer closerS()
+	status := meterStatusDoc{
+		Code:      meterString[MeterNotSet],
```

Comment:
in case where the model meter status is not know (eg. we have not run a migration script) we return a "NOT SET" meter status. Should the meter status change during the lifetime of the metrics manager object, the changed meter status gets written (see LastSuccessfulSend, SetGracePeriod and IncrementConsecutiveErrors).